package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lento",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
Please revert this, the version should not be bumped in pull requests.

Reply:
Done